crates/cxx-qt-build/src/lib.rs

```diff
@@ -291,6 +305,16 @@ pub struct CxxQtBuilder {
     qt_modules: HashSet<String>,
     qml_modules: Vec<OwningQmlModule>,
     cc_builder: cc::Build,
+    extra_defines: HashSet<String>,
+}
+
+/// Options for external crates to use
+#[derive(Default)]
+pub struct CxxQtBuildersOpts {
```

Comment:
As this is likely going to be used by other downstream dependencies as well, we should properly encapsulate this struct, so that we can extend it in future.

I suggest to not make the members public, and add functions to add defines and headers in a builder style.

Reply:
Done :-)